src/main/java/jenkins/branch/BranchIndexingCause.java

```diff
@@ -42,7 +42,7 @@
     }
 
     /**
-     * @inheritDoc
+     * {@inheritDoc}
```

Comment:
Left over from #42 but was blocking me from trying `mvn clean install` in this branch. Note that `BranchIndexingCause` is used by the newly added code.